src/Mod/Arch/ArchCutPlane.py

```diff
@@ -1,26 +1,24 @@
-#***************************************************************************
-#*                                                                         *
-#*   Copyright (c) 2014                                                    *
-#*   Jonathan Wiedemann <wood.galaxy@gmail.com> (cutplan)                  *
-#*   Jerome Laverroux <jerome.laverroux@free.fr (cutline)                  *
-#*                                                                         *
-#*   This program is free software; you can redistribute it and/or modify  *
-#*   it under the terms of the GNU Lesser General Public License (LGPL)    *
-#*   as published by the Free Software Foundation; either version 2 of     *
-#*   the License, or (at your option) any later version.                   *
-#*   for detail see the LICENCE text file.                                 *
-#*                                                                         *
-#*   This program is distributed in the hope that it will be useful,       *
-#*   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
-#*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
-#*   GNU Library General Public License for more details.                  *
-#*                                                                         *
-#*   You should have received a copy of the GNU Library General Public     *
-#*   License along with this program; if not, write to the Free Software   *
-#*   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
-#*   USA                                                                   *
-#*                                                                         *
-#***************************************************************************
+#*****************************************************************************
+#*   Copyright (c) 2014 Jonathan Wiedemann <wood.galaxy@gmail.com> (cutplan) *
+#*   Copyright (c) 2014 Jerome Laverroux <jerome.laverroux@free.fr> (cutline)*
```

Comment:
Jerome added his code a few days ago, therefore this should be dated 2019.

Reply:
Fixed in d9222ff

Reply:
Just for reference, it was #2701, commit 43c5f27.